### PR TITLE
Continue implementing the LEad builder and other refactoring over FormBase

### PR DESCRIPTION
**Pull request by JFVF**
Lead -> continue implementing the builder

CrateLead -> modify to test the builder

Opportunity Form extending from FormBase and deleted the initializer

Form Base moved my initializer that it is invoked when creating a object using builder

### CODE REVIEW

**Comment by JFVF** [comment]
also this pull request contains the first version of gradle

File: build.gradle [hunk 1]
```diff
@@ -25,10 +40,65 @@ dependencies {
...
+        includeGroups(groups)
+    }
+
+    systemProperties.put('primaryUserName', primaryUserName)
+    systemProperties.put('primaryUserPassword', primaryUserPassword)
+    systemProperties.put('browser', browser)
+
+    println browser
+
+    task downloadDrivers(type: Copy) {
```
Comment (carledriss) [inline build.gradle:71]: Great

File: src/test/java/com/salesforce/dev/CreateAccount.java [hunk 2]
```diff
@@ -62,7 +61,7 @@ public void setUp() {
 
     }
 
-    @Test
+    @Test(groups = {"Rgression"})
```
Comment (carledriss) [inline src/test/java/com/salesforce/dev/CreateAccount.java:64]: Typo
Regression

**Comment by JFVF** [comment]
that is it wasn't running all test cases :p

**Pull request merged**